lib/PanelTestProvider.jsm

```diff
@@ -60,6 +61,21 @@ const MESSAGES = () => [
     targeting: `!hasAccessedFxAPanel && !usesFirefoxSync && isFxAEnabled == true`,
     trigger: { id: "toolbarBadgeUpdate" },
   },
+  {
+    id: "WNP_PROFILE_AGE",
+    template: "update_action",
+    content: {
+      action: {
+        id: "moments-wnp",
+        data: {
+          url:
+            "https://www.mozilla.org/en-US/firefox/80.0a1/whatsnew/?oldversion=50.0a2",
+          expire: Date.now() + TWO_DAYS,
```

Comment:
In order for this to work with remote settings, you probably want to make this an expire delta and add the `Date.now()` when the pref is set